var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -24,24 +24,78 @@
 ##############################################################################
 from spack import *
 
+import os
+import shutil
 
-class Scr(Package):
+from llnl.util.filesystem import join_path
```

Comment:
This import isn't necessary. `join_path` is already imported by `from spack import *`. To see exactly what comes with that, check out `spack.__all__`.